### ISSUE TITLE
Upstream this into the official authjs repo

### ISSUE THREAD

**Issue by septatrix** [Hebilicious/authjs-nuxt#142]
### Describe the feature

The [documentation of authjs](https://authjs.dev/reference#help-needed) writes:

> In case you are a maintainer of a package that uses @auth/core, feel free to [reach out to Balázs](https://twitter.com/balazsorban44), if you want to collaborate on making it an official package, maintained in our repository. If you are interested in bringing @auth/core support to your favorite framework, we would love to hear from you!

and has your package listed in their table of community supported packages. Have you considered reaching out to the team to get your package upstreamed?

### Additional information

- [X] Would you be willing to help implement this feature?

**Comment by peterbud** [comment]
I think that would make sense. The official repo already has solidjs and sveltekit modules there. I'd love to see the same endorsement for this nuxt module as well.

**Comment by Hebilicious** [comment]
Agreed. I have reached out to their team and we've already discussed this, it should happen eventually. That being said, there's little community support at the moment, so this isn't a priority.
